.github/workflows/tests.yml

```diff
@@ -19,6 +19,10 @@ jobs:
             pip install poetry
             poetry install
 
+      - name: Install 3rd party from apt
+        run: |
+            sudo apt install unar
+
```

Comment:
Rebase onto main, this dependency is now part of the project and can be removed from this commit.